src/resolvers/git.cr

```diff
@@ -15,6 +15,17 @@ module Shards
       "git"
     end
 
+    def self.expand_resolver_url(url, resolver)
+      case resolver
+      when "git"
+        url
+      when "github", "bitbucket", "gitlab"
+        "https://github.com/#{url}.git"
```

Comment:
You sure about that? ;)

Reply:
Totally

Reply:
Seems there are no specs covering bitbucket and gitlab mappings...